requirements-docs.txt

```diff
@@ -1,3 +1,4 @@
 Jinja2<3.1.0
 Sphinx==3.0.2
 sphinx-notfound-page==0.4
+alabaster==0.7.13
```

Comment:
May be worth adding a comment noting this is because support for Sphinx<3.4 was dropped in alabaster 0.7.14, and Sphinx 3.0.2 has an unbounded dependency. Then we'll know when it can be cleaned up if we update Sphinx later.